code/mob/living/critter/humanoid/human.dm

```diff
@@ -0,0 +1,127 @@
+/**
+ * A file for human mob critters
+ *
+ * The idea is to use a temporary mob equip them and copy the appearance for the critter
+ * Inhands are handled on the handholder just set object_for_inhand as an object path.
+ *
+ * Subtypes should always set a human to copy but don't need to set a corpse spawner.
+ *
+ */
+ABSTRACT_TYPE(/mob/living/critter/human)
+/mob/living/critter/human
+	name = "Human critter parent"
+	desc = "You shouldn't see me!"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "m-none"
+	health_brute = 50
+	health_brute_vuln = 1
+	health_burn = 50
+	health_burn_vuln = 1
+	is_npc = TRUE
+	ai_retaliates = TRUE
+	ai_retaliate_patience = 3
+	ai_retaliate_persistence = RETALIATE_UNTIL_INCAP
+	ai_type = /datum/aiHolder/wanderer
+	/// What do we spawn when we die should be a human corpse spawner leave null for gibs
+	var/corpse_spawner = null
+	/// Path of a human to copy appearance from
+	var/human_to_copy = null
+
+	New()
+		..()
+		src.steal_appearance(src.human_to_copy)
+		src.update_inhands()
+		src.post_setup()
+
+	setup_healths()
+		add_hh_flesh(src.health_brute, src.health_brute_vuln)
+		add_hh_flesh_burn(src.health_burn, src.health_brute_vuln)
+
+	death(var/gibbed)
+		if (gibbed)
+			return ..()
+		..()
+		if (src.corpse_spawner)
+			var/obj/mapping_helper/mob_spawn/corpse/human/body = new src.corpse_spawner(get_turf(src))
+			body.decomp_stage = DECOMP_STAGE_NO_ROT
+			body.max_organs_removed = 0
+			src.ghostize()
+			qdel(src)
+		else
+			src.gib()
+
+	proc/steal_appearance(var/mob/living/carbon/human/H)
+		if (isnull(H))
+			return
+		var/mob/living/carbon/human/target = new H
+		if (target.l_hand) // don't want artifacts of papers / etc.
+			qdel(target.l_hand)
+		if (target.r_hand)
+			qdel(target.r_hand)
+		src.appearance = target
+		src.overlay_refs = target.overlay_refs?.Copy()
+		qdel(target)
```

Comment:
bear in mind this won't `qdel` because you're storing a ref here (also humans generally don't `qdel` 💀), might be fine for small amounts of these but you might need to consider using a typed cache of human instances if you plan to spawn a lot of these